daemon/daemon.go

```diff
@@ -15,6 +15,9 @@ import (
 	"time"
 
 	"github.com/docker/libcontainer/label"
+	"github.com/docker/libnetwork"
+	"github.com/docker/libnetwork/pkg/netlabel"
```

Comment:
What is the difference between packages in `pkg` and other packages in libnetwork? Looking at the [godoc](http://godoc.org/github.com/docker/libnetwork), it seems these aren't getting indexed correctly.